x-pack/plugins/telemetry_collection_xpack/schema/xpack_root.json

```diff
@@ -0,0 +1,2691 @@
+                      },
+                      "lang": {
+                        "type": "array",
+                        "items": {
+                          "type": "keyword"
+                        }
+                      },
+                      "lines_max": {
+                        "type": "long"
+                      },
+                      "lines_total": {
+                        "type": "long"
+                      },
+                      "chars_max": {
+                        "type": "long"
+                      },
+                      "chars_total": {
+                        "type": "long"
+                      },
+                      "source_max": {
+                        "type": "long"
+                      },
+                      "source_total": {
+                        "type": "long"
+                      },
+                      "doc_max": {
+                        "type": "long"
+                      },
+                      "doc_total": {
+                        "type": "long"
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "flattened": {
+              "properties": {
+                "enabled": {
+                  "type": "boolean"
+                },
+                "available": {
+                  "type": "boolean"
+                },
+                "field_count": {
+                  "type": "long"
+                }
+              }
+            },
+            "watcher": {
+              "properties": {
+                "enabled": {
+                  "type": "boolean"
+                },
+                "available": {
+                  "type": "boolean"
+                },
+                "count": {
+                  "properties": {
+                    "total": {
+                      "type": "long"
+                    },
+                    "active": {
+                      "type": "long"
+                    }
+                  }
+                },
+                "execution": {
+                  "properties": {
+                    "actions": {
+                      "properties": {
+                        "_all": {
+                          "properties": {
+                            "total": {
+                              "type": "long"
+                            },
+                            "total_time_in_ms": {
+                              "type": "long"
+                            }
+                          }
+                        },
+                        "slack": {
+                          "properties": {
+                            "total": {
+                              "type": "long"
+                            },
+                            "total_time_in_ms": {
+                              "type": "long"
+                            }
+                          }
+                        },
+                        "DYNAMIC_KEY": {
```

Comment:
This comment applies to all the usages of `DYNAMIC_KEY` int he `*_root.json` files. Should we be more explicit in these, since we _should_ know in advance all the possible values?

Reply:
cc/ @elastic/infra-telemetry

Reply:
> Should we be more explicit in these, since we should know in advance all the possible values?

+1 for explicit values, if we know them in advance 🙏

Reply:
I'd like some input from the ES team here to confirm if we'll know them in advance